packages/cli/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "accessibility-insights-scan",
-    "version": "2.4.6",
+    "version": "2.4.7",
```

Normally, adding new functionality (which this does) would merit a bump to the minor version instead of the patch version
```suggestion
    "version": "2.5.0",
```

---

updated